README.md

```diff
@@ -1,38 +1,10 @@
-This is the common template for language Packs for the [AnySoftKeyboard](https://github.com/AnySoftKeyboard/AnySoftKeyboard) app for Android devices.
-Each pack can contain and provide multiple keyboards or dictionaries for auto correction.
-Most packs are maintained here as [branches of the repository](https://github.com/AnySoftKeyboard/LanguagePack/branches) and published to Google Play Store and F-Droid repository. There are some packs maintained as community forks, here on GitHub or not open source at all. Some of these are:
 
-* [NEO2](https://github.com/kertase/neo_anysoftkeyboard)
-* [Magyar (Hungarian)](https://github.com/rhornig/anysoftkeyboard-hungarian)
-* [Swiss](https://play.google.com/store/apps/details?id=ch.masshardt.anysoftkeyboard.swisslanguagepack)
-* [Pali (Indian)](https://github.com/yuttadhammo/ask-pali-keyboard)
-* [Malayalam, Kannada and other Indian](https://sriandroid.wordpress.com/)
+# Tifinagh Moroccan Language for AnySoftKeyboard
 
-To start a new pack, follow this checklist:
+<p align="center">
+    <img src="./prt.png" width="300"/>
+</p>
```

Comment:
Can you link the images via Markdown instead of HTML?
`![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)`
(see https://guides.github.com/features/mastering-markdown/)

Reply:
Also, could you place the lan.png and prt.png into https://github.com/AnySoftKeyboard/LanguagePack/tree/master/src/main/play/en-US/listing/phoneScreenshots under new names, so they will be used for the store.

Reply:
And for the app icon, you can use the automatic generation, you'd need to import the flag as svg into https://github.com/AnySoftKeyboard/LanguagePack/blob/master/src/main/svg-png/flag/launcher-xxxhdpi.svg. You could use https://commons.wikimedia.org/wiki/File:Berber_flag.svg for that.